src/bench/base58.cpp

```diff
@@ -7,34 +7,37 @@
 #include "validation.h"
 #include "base58.h"
 
+#include <array>
```

Comment:
`verify_script.cpp` is not in our repo. Is yet another earlier PR missing?

Reply:
added in https://github.com/bitcoin/bitcoin/pull/8873 @codablock backported most of this pr so I assume there was a reason he dropped this file. I don't see any mention of such in https://github.com/dashpay/dash/pull/1849 however. File seems (maybe) witness/segwit dependant